### PR TITLE
264 parametrized junits

### PR DESCRIPTION
**Pull request by kzheludkov**

### CODE REVIEW

File: carrot-core/src/test/java/org/bigbase/carrot/BigSortedMapLargeKVsTest.java [hunk 1]
```diff
@@ -405,22 +370,21 @@ private void undelete(List<Key> keys) {
...
-    List<Key> list = new ArrayList<Key>();
+    List<Key> list = new ArrayList<>();
     int collisions = 0;
     while (numDeleted < num) {
       int i = r.nextInt((int) totalLoaded);
       Key key = keys.get(i);
       long len = map.get(key.address, key.length, valPtr, 0, Long.MAX_VALUE);
       if (len == DataBlock.NOT_FOUND) {
         collisions++;
-        continue;
```
Comment (VladRodionov) [inline carrot-core/src/test/java/org/bigbase/carrot/BigSortedMapLargeKVsTest.java:423]: Why did you remove this line?

Comment (VladRodionov) [reply carrot-core/src/test/java/org/bigbase/carrot/BigSortedMapLargeKVsTest.java:423]: nvm

File: carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+public class CarrotCoreBase {
+
+  protected static Codec codec;
+
+  // static {
+  //  UnsafeAccess.debug = true;
+  // }
+
+  @AfterClass
+  public static void cleanAll() throws IOException {
```
Comment (VladRodionov) [inline carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java:25]: Remove method and add @AfterClass to tearDown

File: carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    // Tear down previous run if it was
+    tearDown();
+  }
+
+  public CarrotCoreBase(Object c) throws IOException {
+    codec = (Codec) c;
+    // Tear down previous run if it was
+    tearDown();
```
Comment (VladRodionov) [inline carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java:33]: remove tearDown and setUp from base class

File: carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java (hunk 2 above)
Comment (kzheludkov) [reply carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java:25]: We need to add @AfterClass it to every inherited class then. I will do.

File: carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java (hunk 3 above)
Comment (kzheludkov) [reply carrot-core/src/test/java/org/bigbase/carrot/CarrotCoreBase.java:33]: Done
